apps/web/src/pages/templates/editor/TemplateEditorPage.tsx

```diff
@@ -90,12 +92,16 @@ export default function TemplateEditorPage() {
         if (template._parentId) {
           navigate(`/templates/edit/${template._parentId}`);
         } else {
-          navigate('/templates/');
+          navigate('/templates');
         }
       }
     }
   }, [environment, template]);
 
+  if (environment && environment?.name === 'Production' && isCreateTemplatePage) {
+    navigate('/templates');
+  }
```

Comment:
this if statement won't allow navigating to the create template page in the production